modules/openapi-generator/src/main/resources/rust/reqwest/api.mustache

```diff
@@ -49,20 +48,17 @@ impl {{{classname}}} for {{{classname}}}Client {
             };
             req_builder = req_builder.query(&[("{{{keyParamName}}}", val)]);
         }
-{{/isKeyInQuery}}{{/isApiKey}}{{/authMethods}}{{/hasAuthMethods}}
+        {{/isKeyInQuery}}{{/isApiKey}}{{/authMethods}}{{/hasAuthMethods}}
 
         if let Some(ref user_agent) = configuration.user_agent {
             req_builder = req_builder.header(reqwest::header::USER_AGENT, user_agent.clone());
         }
 
-        {{#hasHeaderParams}}
-        {{#headerParams}}
+        {{#hasHeaderParams}}{{#headerParams}}
```

Comment:
A minor point - if you leave the mustache tags on their own lines, you'll avoid adding extra blank lines.